core/design/src/main/kotlin/edu/stanford/spezi/core/design/theme/Colors.kt

```diff
@@ -87,7 +88,9 @@ internal val RectangleBlue = Color(0xFFEBF2FC)
 private const val DARK_FACTOR = 0.1f
 private const val LIGHT_FACTOR = 0.9f
 
-fun Color.lighten(isDarkTheme: Boolean = false): Color {
+@Composable
+fun Color.lighten(): Color {
+    val isDarkTheme = isSystemInDarkTheme()
```

Comment:
the idea to convert the lighten into a composeable and call isSysteminDarkTheme directly by default is great 🚀